tensorboardX/summary.py

```diff
@@ -175,6 +175,38 @@ def make_image(tensor):
                          encoded_image_string=image_string)
 
 
+def video(tag, tensor):
+    tag = _clean_tag(tag)
+    tensor = makenp(tensor, 'VID')
+    tensor = tensor.astype(np.float32)
+    tensor = (tensor * 255).astype(np.uint8)
+    video = make_video(tensor)
+    return Summary(value=[Summary.Value(tag=tag, image=video)])
+
+
+def make_video(tensor):
+    import moviepy.editor as mpy
+    import tempfile
+
+    t, h, w, c = tensor.shape
+
+    # encode sequence of images into gif string
+    clip = mpy.ImageSequenceClip(list(tensor), fps=4)
+    with tempfile.NamedTemporaryFile() as f:
+        filename = f.name + '.gif'
+
+    import sys
+    save_stdout = sys.stderr
+    sys.stderr = open('trash', 'w')
```

Comment:
`moviepy_error.txt` is better than `trash`. btw, I think it's fine to show the error message in console.

Reply:
That was actually a workaround to avoid the tqdm print moviepy introduces when creating the gif. It didn't work btw.